src/Yasumi/Provider/Australia.php

```diff
@@ -76,73 +75,84 @@ public function calculateAustraliaDay()
     {
         $date = new DateTime("$this->year-01-26", new DateTimeZone($this->timezone));
 
-        $this->calculateHoliday('australiaDay', [], $date);
+        $this->calculateHoliday('australiaDay', ['en_AU' => 'Australia Day'], $date);
     }
 
     /**
      * Function to simplify moving holidays to mondays if required
      *
-     * @param string    $shortName
-     * @param array     $names
-     * @param \DateTime $date
-     * @param bool      $moveFromSaturday
-     * @param bool      $moveFromSunday
+     * @param string          $shortName
+     * @param array           $names
+     * @param string|DateTime $date
```

Comment:
Please use the DateTime type only.

Reply:
Okay, that... breaks a lot. It will take me a few hours to make all the other changes to stop using strings everywhere.